packages/evolution-frontend/src/components/admin/validations/InterviewMap.tsx

```
@@ -1,150 +1,504 @@
+                            height: '40px',
+                            fontSize: '14px',
+                            color: '#fff',
+                            marginBottom: '10px',
+                            boxShadow: '0 2px 4px rgba(0,0,0,0.2)',
+                            gap: '4px'
+                        }}
+                        title={t(
+                            'admin:UndoDrag',
+                            `Undo last drag operation (${dragHistory.length} operation${dragHistory.length !== 1 ? 's' : ''} to undo)`
+                        )}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)